customTests/azure_gpu_vbios.nhc

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+function check_vbios_version() {
+    expected_version="$1"
+    uniq_vbios_versions=$(nvidia-smi -q | grep "VBIOS Version" | cut -d ':' -f 2 | sed 's/ //g' | uniq)
+
+    if [ ${#uniq_vbios_versions[@]} -ne 1 ]; then
+        die 1 "$FUNCNAME: More than 1 VBIOS version found on GPUs! Found '${uniq_vbios_versions[@]}' but expected just '$expected_version'"
```

Comment:
Don't need this part of the error message "More than 1 VBIOS version found on GPUs!"